testutil/discordsession.go

```diff
@@ -72,14 +73,15 @@ func NewInMemoryDiscordSession() *InMemoryDiscordSession {
 }
 
 // ChannelMessageSend records a new message delivery.
-func (s *InMemoryDiscordSession) ChannelMessageSend(channel, message string) (*discordgo.Message, error) {
+func (s *InMemoryDiscordSession) ChannelMessageSend(channel string, message string, options ...discordgo.RequestOption) (*discordgo.Message, error) {
```

Comment:
Similar comment here, re: signature

Reply:
addressed in https://github.com/jakevoytko/crbot/pull/63/commits/8ec82d9359187c8e8a4285d08662bd7cf3cee33d